app/services/HistoricalReturnSubmitService.scala

```diff
@@ -95,15 +95,15 @@ class HistoricalReturnSubmitServiceImpl @Inject()(
 
   def transfer(): Future[Any] = {
 
-    if(appConfig.coreVatReturnsEnabled) {
+    if(appConfig.historicCoreVatReturnsEnabled) {
 
       logger.debug("Starting to process all historical returns to send to core")
```

Comment:
can make this log info add the configured periods?

Can you also add the periods to the log on line 111 https://github.com/hmrc/one-stop-shop-returns/pull/63/files#diff-360b9cf162ce776648226fabcc6cebad01f5a1f9cce4b6da63f4bb12158a6263R111

Reply:
done now